frontend/src/scenes/ingestion/ingestionLogic.ts

```diff
@@ -113,6 +132,12 @@ export const ingestionLogic = kea<ingestionLogicType>([
                 return featFlags[FEATURE_FLAGS.ONBOARDING_1] === 'test'
             },
         ],
+        onboardingSidebar: [
+            () => [],
+            () => {
+                return false
```

Comment:
I assume we want this to be behind a feature flag?

Reply:
Yeah though I think it depends on how we want to implement the next experiment with 1.5 so I've been holding off on turning it into a feature flag just yet

Reply:
yep